gt/main.c

```diff
@@ -93,24 +96,36 @@ gt_parse_incoming_pkt(struct rte_mbuf *pkt, struct gt_packet_headers *info)
 			return -1;
 
 		outer_ipv4_hdr = (struct ipv4_hdr *)info->outer_l3_hdr;
-		parsed_len += sizeof(struct ipv4_hdr);
+		outer_ipv4_hdr_len = ipv4_hdr_len(outer_ipv4_hdr);
+		parsed_len += outer_ipv4_hdr_len;
```

Comment:
We don't need variable `outer_ipv4_hdr_len`, just call `ipv4_hdr_len(outer_ipv4_hdr)` instead.